python/lsst/daf/butler/registry/interfaces/_database.py

```diff
@@ -1763,7 +1774,9 @@ def query(
             connection = self._engine.connect()
         else:
             connection = self._session_connection
-        result = connection.execute(sql, *args, **kwargs)
+        # TODO: SelectBase is not good for execute(), but it used everywhere,
+        # e.g. in daf_relation. We should switch to Executable at some point.
```

Comment:
Can you make a ticket for this if daf_relation is using some old syntax? cc/ @TallJimbo

Reply:
I made DM-37971 for @TallJimbo, but you can reassign it to me if you don't have time to work on that.